api/deployments/component_handler.go

```diff
@@ -261,6 +261,21 @@ func getReplicaSummaryList(pods []corev1.Pod) []deploymentModels.ReplicaSummary
 	return replicaSummaryList
 }
 
+func runningReplicaIsOutdated(component v1.RadixDeployComponent, actualPods []corev1.Pod) bool {
+	// Check if running component's image is not the same as active deployment image tag and that active rd image is equal to 'starting' component image tag
+	componentIsInconsistent := false
+	for _, pod := range actualPods {
+		for _, container := range pod.Spec.Containers {
+			if container.Image != component.Image {
```

Comment:
There is a small time span (I have seen anything from a few seconds up to 20-30 seconds) where the new pod is running and the old pod is terminating.
Will this transition period cause this function to return a false positive? Perhaps there should be an additional condition to only check pods that are not termintating.

Reply:
Should it be continue instead of return false so that all pods in actualPods slice is checked?